CHANGELOG.md

```diff
@@ -2,6 +2,24 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### [3.0.1](https://www.github.com/microsoftgraph/microsoft-graph-toolkit/compare/root-v3.0.0...root-v3.0.1) (2023-07-18)
```

Comment:
This is generated incorrectly and should not have the `root-` prefixes on in the tag names

Reply:
Same thing happens to the PR content that has root: 3.0.1 as the top element. Do we need that?